### DISCUSSION TITLE
Codespace beta glitch

### DISCUSSION THREAD

**Discussion by XxMicrowavexX** [community/community#3883] [Codespaces] [answered]
so i have signed up for beta code spaces I while ago. Today (3rd May) i check the website and it says I haven't signed up but I have! Even when I click on request early access it says I'm on the waiting list!!!

**Comment by PranamBhat** [comment, marked as answer]
Seems like session timed out. Kindly log-out, clear browser history and try again.

**Reply by 2percentsilk** [reply]
Indeed - it's likely your GitHub session expired. If you're on the waiting list, we just ask that you be patient!
